docs/reference/aggregations/bucket/frequent-items-aggregation.asciidoc

```diff
@@ -0,0 +1,177 @@
+[[search-aggregations-bucket-frequent-items-aggregation]]
+=== Frequent items aggregation
+++++
+<titleabbrev>Frequent items</titleabbrev>
+++++
+
+A bucket aggregation which finds frequent item sets. Finding frequent item sets 
+is a form of association rules mining that identifies items that often occur 
+together. Items that are frequently purchased together or log events that tend 
+to co-occur are examples of frequent item sets. Finding frequent item sets helps 
+to discover relationships between different data points (items).
+
+The aggregation reports closed item sets. A frequent item set is called closed 
+if no superset has the same support. For example, we have the two following 
+candidates for a frequent item set **with the same support value**:
+1. `apple, orange, banana`
+2. `apple, orange, banana, tomato`.
+Only the second item set (`apple, orange, banana, tomato`) is returned, and the 
+first set – wich is a subset of the second one – is skipped. Both item sets 
+might be returned if their support values are different. Refer to 
+<<frequent-items-minimum-support,this section>> to learn more about support.
+
+
+==== Syntax
+
+A `frequent_items` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+"frequent_items": {
+  "minimum_set_size": 3,
+  "fields": [
+    {"field": "my_field_1"},
+    {"field": "my_field_2"}
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+.`frequent_items` Parameters
+|===
+|Parameter Name |Description |Required |Default Value
+|`fields` |(array) Fields to analyze. | Required |
+|`minimum_set_size` | (integer) The <<frequent-items-minimum-set-size,minimum size>> of one item set. | Optional | `1`
+|`minimum_support` | (integer) The <<frequent-items-minimum-support,minimum support>> of one item set. | Optional | `0.1`
+|`size` | (integer) The number of top item sets to return. | Optional | `10`
+|===
+
+[discrete]
+[[frequent-items-minimum-set-size]]
+==== Minimum set size
+
+The minimum set size is the minimum number of items the set needs to contain. A 
+value of 1 returns the frequency of single items. The higher the minimum set 
+size the less items are returned. For example, the item set `orange, banana, 
+apple` is only returned if the minimum set size is 3 or lower.
+
+[discrete]
+[[frequent-items-minimum-support]]
+==== Minimum support
+
+Support represents the frequency of an item set normalized between 0 
```

Comment:
Related to https://github.com/elastic/elasticsearch/pull/86037#discussion_r857348240, this is the way I had to wrap around the concept to make sense in my own mind. Hope it's useful:
```suggestion
The minimum support value is the ratio of documents that an item set must exist in to be considered "frequent". In particular, it is a normalized value between 0
```